### PR TITLE
refactor: unify `/detail` page to `pages/_prefix/detail/_id/index.vue`

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] related with https://github.com/kodadot/nft-gallery/issues/4744. remove some `idb-keyval` on `GalleryItem`
- [x] unify `/detail` page to support multi chains with new design

### CODE REVIEW

**Comment by vikiival** [comment]
Oof this is like 3 separated PRs

**Comment by vikiival** [comment]
Maybe I am missing something

@preschian why were Available actions removed?

File: components/rmrk/Gallery/History.vue [hunk 1]
```diff
@@ -377,7 +377,8 @@ export default class History extends mixins(
       event['Block'] = String(newEvent['blockNumber'])
 
       // ID for b-table: Use a unique key of your data Object for each row.
-      event['ID'] = newEvent['timestamp'] + newEvent['id']
+      event['ID'] =
```
Comment (vikiival) [inline components/rmrk/Gallery/History.vue:380]: In Squid Event id is
`export const eventId = (id: string, event: Action) => `${id}-${event}${nanoid()}``

**Review by vikiival — CHANGES_REQUESTED** [review]
Need clarification on removal

File: pages/snek/detail/_id.vue [hunk 2]
```diff
@@ -1,28 +0,0 @@
...
-  },
-  head() {
-    return {
-      link: [
-        {
-          hid: 'canonical',
-          rel: 'canonical',
-          href: this.$root.$config.public.baseUrl + this.$route.path,
-        },
-      ],
```
Comment (Jarsen136) [inline pages/snek/detail/_id.vue:23]: I wonder how could we handle the head config then?

Comment (roiLeo) [reply pages/snek/detail/_id.vue:23]: `useNuxt2Meta`? and after Nuxt3 migration [useHead](https://nuxt.com/docs/getting-started/seo-meta)

**Review by roiLeo — COMMENTED** [review]
💣 time to nuke some pages, code lgtm

but we still have two times the same page:
- `_prefix/gallery/_id/index`
- `_prefix/detail/_id/index` 

Don't we need a fallback if `prefix` doesn't exist?

**Review by vikiival — APPROVED** [review]
Let it roll

**Comment by vikiival** [comment]
pay 45 usd

**Pull request merged**